src/app/components/player/ExtraOptions.tsx

```diff
@@ -92,6 +93,26 @@ const ExtraOptions: React.FC = observer(() => {
             </>
           )}
         />
+        <Toggle
+          checked={player.buffs.thrallSpell}
+          setChecked={(c) => store.updatePlayer({ buffs: { thrallSpell: c } })}
+          label={(
+            <>
+              <img src={thrall.src} width={18} className="inline-block" alt="Thralls" />
+              {' '}
+              Use thralls
+              {' '}
+              <span
+                className="align-super underline decoration-dotted cursor-help text-xs text-gray-300"
+                data-tooltip-id="tooltip"
+                data-tooltip-content="Include thrall damage."
```

Comment:
Could you change this line specifically to
> Include thrall damage, from the Arceuus spellbook.
I think this avoids confusion for people that don't quite know what it is.